src/pipeline.cc

```diff
@@ -1199,6 +1200,25 @@ NAN_METHOD(pipeline) {
   baton->webpLossless = AttrTo<bool>(options, "webpLossless");
   baton->webpNearLossless = AttrTo<bool>(options, "webpNearLossless");
   baton->tiffQuality = AttrTo<uint32_t>(options, "tiffQuality");
+  // tiff compression options
+  std::string tiffCompression = AttrAsStr(options, "tiffCompression");
```

Comment:
Okay yea this looks like a better idea. Could you show me where I define the map between nickname -> type though? I can't seem to find it for the example you gave, so I'm wondering how the function takes the string input and decides which type to use?